apps/service_providers/messaging_service.py

```diff
@@ -0,0 +1,50 @@
+from io import BytesIO
+
+import pydantic
+import requests
+from twilio.rest import Client
+
+from apps.channels import audio
+from apps.channels.models import ChannelPlatform
+
+
+class MessagingService(pydantic.BaseModel):
+    _type: str
+    _supported_platforms: list
```

Comment:
General question. What do you think about moving away from pydantic? I think we could use dataclasses as easily.

Reply:
I have no reservations. We're using pydantic to define the tools' schemata for agents (in the `schemas.py` file). If that can work with dataclasses, I don't see why not. I suspect it wouldn't be an issue